classes/genre.rb

```diff
@@ -0,0 +1,17 @@
+require_relative 'item'
+
+class Genre
+  attr_accessor :name
+  attr_reader :id, :items
+
+  def initialize(name)
+    @id = id
```

Comment:
you should asign a value to @id not just id, use rand(1...1000)

Reply:
Thank You. I am sorry for the error